config/helm/chart/default/templates/Common/csi/csidriver.yaml

```diff
@@ -18,7 +18,7 @@ kind: CSIDriver
 metadata:
   name: csi.oneagent.dynatrace.com
   labels:
-    {{- if (eq (include "dynatrace-operator.openshiftOrOlm" .) "true") }}
+    {{- if eq (include "dynatrace-operator.platform" .) "openshift" }}
```

Comment:
can you please update or add new helm test?

Reply:
The code depends on 3 variables 
- `platform`
- needCSI (`or`): 
  - `csidriver.enabled`
  - `partial`

The test should also set all of them and cover at least 2 basic cases
```
- it: should contain correct labels for openshift with CSI enabled
    set:
      platform: openshift
      csidriver.enabled: true
      partial: false

  - it: should contain correct labels for openshift manifest
    set:
      platform: openshift
      csidriver.enabled: false
      partial: csi
```

Reply:
please add `partial`  to the first case 

```
  - it: should contain correct labels for openshift with CSI enabled
    set:
      platform: openshift
      csidriver.enabled: true
      partial:false                     <----
    asserts:
```

Reply:
thanks!